src/public/app/widgets/type_widgets/type_widget.js

```diff
@@ -38,6 +38,9 @@ export default class TypeWidget extends NoteContextAwareWidget {
         return this.$widget.is(":visible") && this.noteContext?.ntxId === appContext.tabManager.activeNtxId;
     }
 
+    /**
+     * FIXME: add async here to indicate promise?
```

Comment:
With jsdoc we can indicated that either a Promise or the final value can be returned:

```
/**
 * @returns {Promise|*} promise resolving content or directly the content
 */ 
```